build.sh

```diff
@@ -18,4 +18,11 @@ sed -e "/#INSERT_PYTHON_CODE_HERE/r ./src/aws_cli_mfa.py" -e "s///" ./src/$AWS_C
     sudo chmod +x /usr/local/bin/aws-cli-mfa &&
     cp /usr/local/bin/aws-cli-mfa ./bin/aws-cli-mfa-$AWS_CLI_MFA_SHELL
 
+set AWS_CLI_MFA_SHELL fish && \
```

Comment:
This file is running in bash, per the shebang. Please keep fish in here, but use the correct bash syntax.

Reply:
Updated.  Curious why the extra hop in the build script into `/usr/local/bin/aws-cli-mfa` just to copy it into the local `bin` and then remove the copy in `/usr/local/bin`?  Since the README instructions have you copy the file out of the local `bin` back into the original location, should the commands in this script just build the local `bin` files?

Reply:
Hmm could you add a separate commit showing what you are suggesting cleaning up? It sounds intriguing but I'm not sure I follow

Reply:
Here is what I had in mind:
https://github.com/rogusdev/aws-cli-mfa/pull/9/commits/633100121064cdd5525f470c1fb1fdc994f75d4f

It effectively removes an extra copy since at the end of the script we are deleting `/usr/local/bin/aws-cli-mfa` anyway.  Also, it seems to have fixed a bug where `help='do not export/set AWS_PROFILE to the one generated by STS')` was duplicated for `--lifetime-duration` arg as well.